port/common/omrsysinfo.c

```diff
@@ -936,3 +936,50 @@ omrsysinfo_cgroup_is_memlimit_set(struct OMRPortLibrary *portLibrary)
 {
 	return FALSE;
 }
+
+/**
+ * Gets the file pointer by opening the file passed as a param fileName
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ *
+ * @param[in] subsystemFlag for checking the index of the subsystem
+ *
+ * @param[in] fileName pointer to cgroup subsystem filename
+ *
+ * @param[in] file pointer to the file pointer
+ *
+ * @return 0 on success and OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM on failure
```

Comment:
This documentation is out-of-date.

Reply:
Thanks for the review @keithc-ca. Updated the documentation accordingly for the current changes made.